x-pack/plugins/security_solution/server/lib/detection_engine/routes/rules/import_rules_route.test.ts

```diff
@@ -229,6 +241,19 @@ describe('import_rules_route', () => {
       });
     });
 
+    test('returns 200 if many rules are imported successfully', async () => {
+      const ruleIds = new Array(9999).fill(undefined).map((_, index) => `rule-${index}`);
+      const multiRequest = getImportRulesRequest(buildHapiStream(ruleIdsToNdJsonString(ruleIds)));
+      const response = await server.inject(multiRequest, context);
+
+      expect(response.status).toEqual(200);
+      expect(response.body).toEqual({
+        errors: [],
+        success: true,
+        success_count: 9999,
+      });
+    });
```

Comment:
Nice tests, we had a few from a while ago I did remove that were testing 10,001 to hit the error that you cannot have 10,001 but it had slowed down so much I removed it. Which in hindsight was probably a mistake on my side. I wonder if you went to 10,000 here (since it's not an end to end test) if it will give that error and then we have some coverage again for it (or would it timeout?)